LayoutKit.playground/Contents.swift

```diff
@@ -20,3 +20,5 @@ helloWorld.arrangement().makeViews()
 helloWorld.arrangement(width: 250).makeViews()
 
 helloWorld.arrangement().makeViews(direction: .RightToLeft) // just for testing; RTL happens automatically for RTL languages.
+
+CircleImagePileLayout(imageNames: ["earth.png", "earth.png"], mode: .trailingOnTop).arrangement().makeViews()
```

Comment:
do you want to keep this in the example playground or was this just for testing?